hack/run-smoke.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright 2019 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# runs all smoke tests for the operator
+SCRIPT_NAME=`basename $0`
+
+function usage(){
```

Comment:
we could make this a default option for all scripts in this dir.

Reply:
We should open an issue to update the scripts.